docs/meshcloud.customer.md

```diff
@@ -16,11 +16,11 @@ and any additional [metadata specific to your organization](meshstack.metadata-t
 
 ## Managing your meshCustomer
 
-Every aspect of your meshCustomer can be managed in the so-called [customer control plane](./meshcloud.customer.md#managing-your-meshcustomer). This customer control plane looks like the screenshot below.
+Every aspect of your meshCustomer can be managed in the so-called [customer control plane](./meshcloud.customer.md#managing-your-meshcustomer). The customer control plane is the highest control plane. From that level, you can navigate to various control planes like the [project control plane](./meshcloud.project-resources.md#project-control-plane) or the [tenant control plane](./meshcloud.project-resources.md#tenant-control-plane). Each control plane has a similar scheme. Depending on your permissions within the customer and the meshStack configuration, you will have access to different to tabs like **Settings** or **Financials**. The customer control plane below shows the control plane from the perspective of a Customer Admin.
```

Comment:
f: Please remove the second `to`
 `...you will have access to different tabs like **Settings** or **Financials**. ...`